src/Features/Lsif/Generator/Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.csproj

```diff
@@ -44,8 +44,13 @@
                               PackagePath="tools\$(TargetFramework)\any\%(ReferenceCopyLocalPaths.DestinationSubDirectory)"
                               Condition="'%(ReferenceCopyLocalPaths.DestinationSubDirectory)' == '' and '%(Extension)' != '.pdb' and '%(Extension)' != '.xml'" />
 
-      <TfmSpecificPackageFile Include="@(BuiltProjectOutputGroupOutput)" PackagePath="tools\$(TargetFramework)\any" />
-      <TfmSpecificPackageFile Include="$(ProjectDepsFilePath)" PackagePath="tools\$(TargetFramework)\any" Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'" />
+      <!-- Include content output as well, for the BuildHost -->
```

Comment:
💭 It's hard for me to compare this to another project where we did similar:
https://github.com/sharwell/JackFruit/blob/64002cbcabf67f42684515c7a2633bc63c06a2f0/Jackfruit.AppModel.Package/Jackfruit.AppModel.Package.csproj

Reply:
Did you have a specific concern?